server/templates/delete_students.html.j2

```diff
@@ -9,7 +9,9 @@
       {{ form.emails(class="mdl-textfield__input", type="text", rows="10", id="delemails") }}
       <label class="mdl-textfield__label" for="delemails">Emails...</label>
     </div>
-
+    <div class="checkbox" id="use_all_emails">{{ form.use_all_emails }}
```

Comment:
Newline before the {{

...the HTML is super confusing with functions just for the input but not the label